docs/.sphinx/wordslist.txt

```diff
@@ -1,83 +1,82 @@
 APIs
-BNF
-Baz
-CLI
-Configs
-DMI
-DNS
-DUT
-ESC
-Fi
-FooBar
-GPLv
-Github
-INI
-IQN
-Instantiation
-JSON
-Jinja
-Naur
-Parsers
-Permalink
-PlainBox
-Plainbox
-PyPI
-QML
-RTC
-SIGINT
-SRU
-Snapcraft
-Systemd
-UI
-Wi
-XSLX
 autoconf
 baz
+Baz
 bluetooth
+BNF
 boolean
 chroot
 cli
+CLI
 commandline
 config
 configs
+Configs
 cpu
 ctrl
 de
 debhelper
 defs
 desc
+DMI
+DNS
+DUT
 enquote
+ESC
+Fi
+FooBar
 formatter
 github
+Github
```

Comment:
If casing anything, surely then `GitHub`?

Reply:
Will do.

(for info, these terms show up because I sorted the wordslist in this PR, not because I've actually added these)